src/main.R

```diff
@@ -0,0 +1,28 @@
+library(dplyr)
+
+source(here::here("src/data_processing.R"))
+source(here::here("src/auto_scaling_algorithm.R"))
+source(here::here("src/calculate_adi.R"))
+
+args = commandArgs(trailingOnly=TRUE)
+input_data <- read.csv(args[1])
+util_data <- processing_data("data/util_data.csv")
+
+# Test parameters
+initial_allocated_cores <- 2
```

Comment:
shouldn't we assign below hardwired values via args or config?

Reply:
Added config file